Resources/doc/configuration.rst

```diff
@@ -1003,33 +1003,48 @@ can configure. The following block shows all possible configuration keys:
                 http://symfony.com/schema/dic/doctrine/doctrine-1.0.xsd">
 
             <doctrine:config>
+                <!-- path: SQLite specific -->
+                <!-- memory: SQLite specific -->
+                <!-- multiple-active-result-sets: driver specific -->
+                <!-- pooled: Oracle specific (SERVER=POOLED) -->
+                <!-- protocol: IBM DB2 specific (PROTOCOL) -->
+                <!-- server: SQL Anywhere specific (ServerName) -->
+                <!-- service: Oracle specific (SERVICE_NAME instead of SID) -->
+                <!-- servicename: Oracle specific (SERVICE_NAME) -->
+                <!-- sessionMode: oci8 driver specific (session_mode) -->
+                <!-- default_dbname: PostgreSQL specific (default_dbname) -->
+                <!-- sslmode: PostgreSQL specific (LIBPQ-CONNECT-SSLMODE) -->
+                <!-- sslrootcert: PostgreSQL specific (LIBPQ-CONNECT-SSLROOTCERT) -->
+                <!-- sslcert: PostgreSQL specific (LIBPQ-CONNECT-SSLCERT) -->
+                <!-- sslkey: PostgreSQL specific (LIBPQ-CONNECT-SSLKEY) -->
+                <!-- sslcrl: PostgreSQL specific (LIBPQ-CONNECT-SSLCRL) -->
```

Comment:
How about grouping things that go together in one comment?
```
                <!-- 
                      sslmode: PostgreSQL specific (LIBPQ-CONNECT-SSLMODE)
                      sslrootcert: PostgreSQL specific (LIBPQ-CONNECT-SSLROOTCERT)
                      sslcert: PostgreSQL specific (LIBPQ-CONNECT-SSLCERT)
                      sslkey: PostgreSQL specific (LIBPQ-CONNECT-SSLKEY)
                      sslcrl: PostgreSQL specific (LIBPQ-CONNECT-SSLCRL)
                -->
```

Reply:
Good idea!